oss-internship-2020/libtiff/README.md

```diff
@@ -0,0 +1,22 @@
+# sapi-libtiff
+Copyright 2020 Google LLC.
+
+## Start use
+You should make sure the libtiff submodule is cloned.
+
+`git clone --recursive https://github.com/alexelex/sapi-libtiff`
+
+## Usage
+
+#### build:
+`mkdir -p build && cd build && cmake .. -DSAPI_ROOT=$HOME/sapi_root -DBUILD_SHARED_LIBS=OFF`
+
+`make -j 8`
+
+#### to run the sandboxed example:
+`./example/sandboxed absolute/path/to/project/dir`
```

Comment:
`absolute/path/to/project/dir` is not an absolute path and sends mixed signals, use `/absolute/path/to/project/dir` instead (note the first `/`)